src/CountryHandler/CountryHandler.php

```diff
@@ -33,11 +33,7 @@ final public function __construct(string $tin = '')
 
     public function getTIN(): string
     {
-        if (null !== $string = preg_replace('#[^[:alnum:]\-+]#u', '', $this->tin)) {
-            return strtoupper($string);
-        }
-
-        return '';
+        return strtoupper($this->tin);
```

Comment:
I would prefer to keep that in the existing normalize function. WDYT?

Reply:
I agree you